### ISSUE TITLE
IO Error: No such file or directory (os error 2)

### ISSUE THREAD

**Issue by DimitriPapadopoulos** [neurospin/neuro-forge#19]
`neuro-forge` exits with uncaught exception:
```
$ neuro-forge build /tmp/channel spm12
[...]
 ╭─ Finding outputs from recipe
 │ Found 1 variants
 │ Build variant: spm12-7771-hb0f4dca_0
 │ 
 │ ╭─────────────────┬──────────╮
 │ │ Variant         ┆ Version  │
 │ ╞═════════════════╪══════════╡
 │ │ target_platform ┆ linux-64 │
 │ ╰─────────────────┴──────────╯
 │
 ╰─────────────────── (took 0 seconds)
Error:   × IO Error: No such file or directory (os error 2)
  ╰─▶ No such file or directory (os error 2)

Traceback (most recent call last):
  File "/volatile/src/neuro-forge/.pixi/envs/default/bin/neuro-forge", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/volatile/src/neuro-forge/.pixi/envs/default/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
[...]
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['env', 'HOME=/tmp/channel', 'rattler-build', 'build', '-r', '/volatile/src/neuro-forge/recipes/spm12', '--output-dir', '/tmp/channel', '--experimental', '-c', 'conda-forge', '-c', 'bioconda']' returned non-zero exit status 1.
```

**Comment by sapetnioc** [comment]
The OS error comes from rattler-build command. To make this more clear, a simple message is written now instead of a Python exception.

**Comment by sapetnioc** [comment]
I made the change in the spm12 branch.

**Comment by DimitriPapadopoulos** [comment]
Please apply your changes to the `main` branch instead, I cannot rebase.

**Comment by DimitriPapadopoulos** [comment]
I have moved your changes to branch [`recipes-directory`](https://github.com/neurospin/neuro-forge/tree/recipes-directory).

**Comment by sapetnioc** [comment]
Fixed with #20
